test/geometry/LinearPropagator.test.cu

```diff
@@ -58,7 +58,7 @@ __global__ void linProp_test_kernel(const GeoParamsPointers shared,
         distances[tid.get() * max_segments + seg] = geo.next_step();
 
         // Move next step
-        propagate();
+	auto temp = propagate();
```

Comment:
Could you add result data (pointers to `VolumeId` and `double`) to check the result from the propagation?